locales/en/cos-ui.json

```diff
@@ -37,6 +38,12 @@
   "clusterName": "Cluster name",
   "clusterError": "Could not get the cluster name",
   "common": "Common",
+  "communicationErrorDescripton": "Error: \"API message\"",
+  "communicationErrorEmail": "rhosak-eval support@redhat.com",
```

Comment:
It looks like there's a typo in the email address
Instead of this:
> rhosak-eval support@redhat.com

It should include an extra dash (-) like this:
> rhosak-eval-support@redhat.com

Reply:
@Kells512 also no need to add the email id for the i18n string. instead, add this to the constant file [here](https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/main/src/constants/constants.tsx)